vta/apps/serialLoadMM/README.md

```diff
@@ -0,0 +1,42 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+VTA TSIM Application 
+======================
+Prior to this application, please take a look at `<tvm-root>/vta/apps/tsim_example` for installation
+This is an application that performs Bit Serial Multiplication for GEMM utilizing TSIM.
+
+* Test Chisel3 backend
+    * Go to `<tvm-root>/vta/apps/serialLoadMM`
+    * Run `make`
+
+* Testing on compiled backend
+    * If you have already compile chisel backend and want test with another input set, run `make test`
```

Comment:
Can we give instructions on how to use this app as (1) bit serial GEMM, and (2) traditional bit-parallel GEMM?

Reply:
I think that would be good to show the dual purpose of the circuit presented here

Reply:
Added bit parallel test script option for user